modules/admediaBidAdapter.js

```diff
@@ -0,0 +1,102 @@
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import {BANNER} from '../src/mediaTypes.js';
+
+const BIDDER_CODE = 'admedia';
+const ENDPOINT_URL = 'https://prebid.admedia.com/bidder/';
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER],
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {BidRequest} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function (bid) {
+    return !!(bid.params.placementId);
+  },
+
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @return Array Info describing the request to the server.
+   * @param validBidRequests
+   * @param bidderRequest
+   */
+  buildRequests: function (validBidRequests, bidderRequest) {
+    if (validBidRequests.length === 0) {
+      return [];
+    }
+    return validBidRequests.map(bidRequest => {
+      let sizes = []
+      if (bidRequest.mediaTypes && bidRequest.mediaTypes[BANNER] && bidRequest.mediaTypes[BANNER].sizes) {
+        sizes = bidRequest.mediaTypes[BANNER].sizes;
+      }
+
+      var tagData = [];
+
+      var tagObjects = [];
+      tagObjects['tags'] = [];
```

Comment:
What is the purpose of this? This doesnt seem to be referenced here and will always be returned as an empty array.

Reply:
Thanks @mmoschovas , removed it as not needed.